docs/source/peers/peers.md

```diff
@@ -341,6 +370,13 @@ application and a set of peers --- it does not involve orderers. Phase 1 is only
 concerned with an application asking different organizations' endorsing peers to
 agree to the results of the proposed chaincode invocation.
 
+**Attention**: Hyperledger Fabric v2.4 introduces Fabric Gateway, which
+simplifies client application development by migrating transaction proposal and
+endorsement code from the client application to the gateway peer. Refer to
+the [Fabric Gateway](../gateway.html) documentation on the gateway process
+and simplified SDKs. The legacy process, including coding transaction proposal and
+endorsement in the client application, remains supported, as discussed below.
+
```

Comment:
Instead of just adding the Attention note, I thought we would we would be updating the text throughout to assume that gateway manages the endorsements and transaction submission, rather than the client application managing the endorsements and transaction submission.  We can say that client applications using the legacy SDKs can alternatively manage the endorsements, but I think the main flow of the transaction process in these sections should be based on the gateway now.

The problem with just adding Attention notes is two-fold:
- We want new readers to think of the gateway flow as the default flow.
- We don't want to have to update these docs yet again in a year to make the changes to the main text. Let's just do it now and be done with it.

Reply:
It's fair, it's substantially more rewriting and review. The issue is the old way is still supported, so I'll give it a go to show that both models work.